tests/util.go

```diff
@@ -12,9 +12,23 @@ import (
 	ktests "kubevirt.io/kubevirt/tests"
 )
 
+type Result struct {
+	verb          string
+	resourceType  string
+	resourceName  string
+	resourceLabel string
+	filePath      string
+	nameSpace     string
+	query         string
+	expectOut     string
+}
+
 var KubeVirtOcPath = ""
 
 const (
+	CDI_LABEL_KEY          = "app"
+	CDI_LABEL_VALUE        = "containerized-data-importer"
+	CDI_LABEL_SELECTOR     = CDI_LABEL_KEY + "=" + CDI_LABEL_VALUE
```

Comment:
CDI related const should not go util, it should put into your cdi_test.go

Reply:
thanks for the suggestion, I will considering moving those constants into somewhere else in a follow-up pr.